docs/pages/application-access/guides/vnet.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: VNet
+description: How to configure custom DNS zones for VNet
+---
+
+VNet automatically proxies connections made to TCP applications available under the public address
+of a Proxy Service. This guide explains how to configure VNet to support apps with [custom public
+addresses](connecting-apps.mdx/#customize-public-address). We are going to take an app available at
+`tcp-app.teleport.example.com` and make it available through VNet at <Var name="public_addr" initial="tcp-app.company.test"/>
+with <Var name="suffix" initial="company.test" /> as the custom DNS zone.
```

Comment:
I'm not sure if it's fine to use `<Var />` within the introductory paragraph like this. I haven't really seen that anywhere else, maybe other than in [Helm AWS docs](https://goteleport.com/docs/deploy-a-cluster/helm-deployments/aws/). But even there it's not at the very top and has its own admonition box.